app/features/app/components/App.js

```diff
@@ -3,8 +3,23 @@
 import React, { Component } from 'react';
 
 import { Conference } from '../../conference';
+import { Welcome } from '../../welcome';
+
 import config from '../../config';
 
+// $FlowFixMe
+import styled from 'styled-components';
+
+const Container = styled.div`
```

Comment:
Yeah. I just placed them there so I can check if they work. 
They definitely need some organisation.

> They have styled components in a styled.js file or in individual files inside a styled directory if need be. I think we can just go with the single file inside each feature.

It will be good either way. But I think we should go with directory. They will be much more organised and will be better if we have more number of styled components. (Thinking about future too)

> When a component "augments" the styled thing, the styled object is called FooElement and the component is called Foo, which renders a FooElement, among doing other things.

I actually think this has benefits too.
But they are same as react components which can be placed in `components` directory of `features`. If we need styled components in it. we can import from `styled` directory.

Reply:
Makes sense, let's do this!